widget/scroller.go

```diff
@@ -371,10 +373,20 @@ func (s *ScrollContainer) Dragged(e *fyne.DragEvent) {
 
 // MinSize returns the smallest size this widget can shrink to
 func (s *ScrollContainer) MinSize() fyne.Size {
+	if s.minSize.Width > 0 && s.minSize.Height > 0 {
```

Comment:
```suggestion
	if if s.minSize != (fyne.Size{}) {
```

Reply:
second try :)

Reply:
Oh, good catch! is the deliberate mistake there just to keep me on my toes? ;)

Reply:
Ehm … yes, yes of course :).

Reply:
Fixed the proposal.

Reply:
Silly me, I should read ahead.